conf/dev.conf

```diff
@@ -25,6 +25,7 @@ topics {
       min-back-off = 5.millis
       max-back-off = 10.millis
       max-retries = 5
+      path = "$.event[?(@.enrolment =~ /HMRC\\-CUS\\-ORG\\~EORINumber~.*/i)]"
```

Comment:
I would've gone for something like "jsonFilter" or "jsonPathFilter".. so it's clear to people who don't read the readme that it's a filtering predicate in its entirety

Reply:
Elie I dont mind changing it , but in story path is mentioned everywhere